dds/idl/metaclass_generator.cpp

```diff
@@ -40,14 +40,17 @@ bool metaclass_generator::gen_enum(AST_Enum*, UTL_ScopedName* name,
 {
   ContentSubscriptionGuard csg(!be_global->v8());
   NamespaceGuard ng;
-  std::string decl = "const char* gen_" + scoped_helper(name, "_") + "_names[]";
-  be_global->header_ << "extern " << decl << ";\n";
-  be_global->impl_ << decl << " = {\n";
+  std::string array_decl = "const char* gen_" + scoped_helper(name, "_") + "_names[]";
+  std::string size_decl = "const CORBA::ULong gen_" + scoped_helper(name, "_") + "_names_size";
```

Comment:
Is there a reason to use `CORBA::ULong` here?  It's not really tied to the IDL mapping, so could it just be `std::size_t` to match C++ `sizeof`?

Reply:
OK